apps/web/src/components/templates/email-editor/EmailMessageEditor.tsx

```diff
@@ -81,33 +60,26 @@ export function EmailMessageEditor({
   }
 
   function onBlockAdd(type: EmailBlockTypeEnum) {
-    const modifiedBlocks = [...blocks];
-
     if (type === 'button') {
-      modifiedBlocks.push({
+      contentBlocks.append({
```

Comment:
I favour to pass the property as argument and not to do these by reference modifications. For me they are more prone to hard to debug bugs. But it is a personal preference. Also I noticed it was implemented like this already. 👍🏻